### DISCUSSION TITLE
gpt-4o-mini 速度问题

### DISCUSSION THREAD

**Discussion by imoyy** [BurnHair/docs#179] [分享讨论]
如图，4o的反应速度明显快于4o-mini。在实际使用时也是相同的体验，本站的4o-mini的响应时间比其它模型长好多。
这是我在美国服务器上部署的oneapi，是因为地区问题还是因为其它？后续是否会优化？

![CleanShot 2024-08-02 at 16 46 18@2x](https://github.com/user-attachments/assets/b41b2439-a594-4d57-80c7-cfb931482943)
![CleanShot 2024-08-02 at 16 45 58@2x](https://github.com/user-attachments/assets/3bf15130-0c0b-44dd-80c3-92142bee8a6e)

**Comment by BennyThink** [comment]
嗯，因为之前 4o-mini 还没正式GA，所以用的是其他的渠道。现在正式GA了，所以最近会优化的。

**Comment by BennyThink** [comment]
做了一点小优化，你试试看 :-)

**Reply by imoyy** [reply]
> 做了一点小优化，你试试看 :-)

已经ok了
![CleanShot 2024-08-02 at 17 54 51@2x](https://github.com/user-attachments/assets/cb7d80ac-e7e0-4583-aff4-272e439ca5d5)
